presto-main/src/main/java/com/facebook/presto/sql/analyzer/ExpressionAnalyzer.java

```diff
@@ -226,7 +226,7 @@ public Type setExpressionType(Expression expression, Type type)
 
     public IdentityLinkedHashMap<Expression, FieldId> getColumnReferences()
     {
-        return new IdentityLinkedHashMap<>(columnReferences);
+        return columnReferences;
```

Comment:
No for two reasons: consistency (see other getters ther), and with `unmodifiableMap` I can no longer  `IdentityLinkedHashMap` return type. This is dirty, but it will be dirty only until we have `Map<NodeRef,...>`s